rules/architecture-diagram/rule.md

```diff
@@ -164,6 +164,19 @@ Maintain standards to keep your diagrams consistent:
 ![Figure: Good Example - SSW People (a Static Site - Gatsby and React with Dynamics 365 and SharePoint Online) - you can just as easily create colorful, engaging diagrams suitable for all of your project stakeholders](SSW.People-Architecture-Diagram.png)
 :::
 
+### Tip 9: Tip 9 – Where to store Diagrams?
+
+Standardizing where your organisation stores architecture diagrams is important to ensure a consistent experience among developers. There are a few options depending on which source control you use:
+
```

Comment:
As per our rule (https://www.ssw.com.au/rules/awesome-documentation) the project architecture image is usually in the README.md file which will be located in root folder of the repo.

The rest of the documentation goes into a `/docs` folder - regardless of which Source Control is being used.

When using Azure DevOps I prefer the coded wiki which gets pulled down with the code from the repository. The other option is to use the built-in wiki that does not get pulled down with the code from the repository.

When using GitHub, there is also a separate Wiki repo for your project. 
For instance to clone your wiki to your local machine you can do: `git clone https://github.com/YOUR_USERNAME/YOUR_REPOSITORY.wiki.git`
Notice the `.wiki.` part of the git link.